boards/arm/stm32l476g_disco/doc/stm32l476g_disco.rst

```diff
@@ -0,0 +1,231 @@
+    - 2x SAIs (serial audio interface)
+    - 3x I2C FM+(1 Mbit/s), SMBus/PMBus
+    - 6x USARTs (ISO 7816, LIN, IrDA, modem)
+    - 3x SPIs (4x SPIs with the Quad SPI)
+    - CAN (2.0B Active) and SDMMC interface
+    - SWPMI single wire protocol master I/F
+- 14-channel DMA controller
+- True random number generator
+- CRC calculation unit, 96-bit unique ID
+- Development support: serial wire debug (SWD), JTAG, Embedded Trace Macrocell |trade|
+
+
+More information about STM32L476VG can be found here:
+       - `STM32L476VG on www.st.com`_
+       - `STM32L476 reference manual`_
+
+
+Supported Features
+==================
+
+The Zephyr stm32l476g_disco board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/stm32l476g_disco/stm32l476g_disco_defconfig``
+
+
+Connections and IOs
+===================
+
+STM32L476G Discovery Board has 8 GPIO controllers. These controllers are responsible for pin muxing,
+input/output, pull-up, etc.
+
+For mode details please refer to `STM32L476G Discovery board User Manual`_.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+- UART_2_TX : PD5
+- UART_2_RX : PD6
+- LD4 : PB2
+- LD5 : PE8
+
+System Clock
+------------
+
+STM32L476G Discovery System Clock could be driven by an internal or external oscillator,
+as well as the main PLL clock. By default the System clock is driven by the PLL clock at 80MHz,
+driven by 16MHz high speed internal oscillator.
+
+Serial Port
+-----------
+
+STM32L476G Discovery board has 6 U(S)ARTs. The Zephyr console output is assigned to UART2.
+Default settings are 115200 8N1.
+
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+
+STM32L476G Discovery board includes an ST-LINK/V2-1 embedded debug tool interface.
+This interface is supported by the openocd version included in Zephyr SDK.
+
+Flashing an application to STM32L476G Discovery
+-----------------------------------------------
+
+The sample application :ref:`hello_world` is being used in this tutorial:
+
+.. code-block:: console
```

Comment:
You should delete this code block (lines 178-181)

Reply:
Done!